MoltenVK/MoltenVK/GPUObjects/MVKFramebuffer.h

```diff
@@ -57,5 +58,7 @@ class MVKFramebuffer : public MVKVulkanAPIDeviceObject {
 	VkExtent2D _extent;
 	uint32_t _layerCount;
 	MVKSmallVector<MVKImageView*, 4> _attachments;
+	bool _imageless;
+	MVKSmallVector<MVKImageView*, 4> _imagelessAttachments;
```

Comment:
Wait, why do we need another vector for the imageless attachments? This isn't even used.